src/pybind/mgr/object_format.py

```diff
@@ -218,4 +218,8 @@ def _format_response(*args: Any, **kwargs: Any) -> Tuple[int, str, str]:
                 return e.format_response()
             return retval, body, ""
 
+        # set the extra args on our wrapper function. this will be consumed by
+        # the CLICommand decorator and added to the set of optional arguments
+        # on the ceph cli/api
+        setattr(_format_response, "extra_args", {"format": str})
```

Comment:
I don't fully understand the question, but I'll try to answer: we add an extra argument named "format" with a value of the "str" type. When the code that processes the extra args runs it will ensure that an argument such as `--format` with strnig values will be accepted by the mgr module before being dispatched to the correct "endpoint".